copier/user_data.py

```diff
@@ -380,7 +391,9 @@ def get_when(self, answers) -> bool:
         when = cast_answer_type(when, cast_str_to_bool)
         return bool(when)
 
-    def render_value(self, value: Any) -> str:
+    def render_value(
+        self, value: Any, answers: Optional[Mapping[str, Any]] = None
```

Comment:
note, only added `answers` here as a way to pass the *already cast* value from `validate_answer`, but if there's a better way to do this, lemme know

Reply:
yeah, I prefer that too thanks. (yep, that's the correct annotation)